requirements/dev.txt

```diff
@@ -4,7 +4,7 @@
 #
 #    make upgrade
 #
--e git+https://github.com/edx/acid-block.git@e46f9cda8a03e121a00c7e347084d142d22ebfb7#egg=acid-xblock
+-e git+https://github.com/edx/acid-block.git@98aecba94ecbfa934e2d00262741c0ea9f557fc9#egg=acid-xblock
```

Comment:
@feanil I have updated the hash here. acid-block is using unicode method in old hash.

Reply:
I have updated this in test files also.

Reply:
[Old hash file ](https://github.com/edx/acid-block/blob/e46f9cda8a03e121a00c7e347084d142d22ebfb7/acid/acid.py#L113)which is causing issue.